controllers/Users.php

```diff
@@ -232,9 +249,7 @@ public function update_onDelete($recordId = null)
 
         Flash::success(Lang::get('backend::lang.form.delete_success'));
 
-        if ($redirect = $this->makeRedirect('delete', $model)) {
-            return $redirect;
-        }
+        return Redirect::refresh();
```

Comment:
You made the change in the wrong method. Redirect::refresh() should be in the unsuspend, everything else should be left alone.

Reply:
My bad, trying to correct things at the end of the day. Double checked and sorted.